packages/@okta/vuepress-site/guides/sign-into-mobile-app/define-callback/ios/definescheme.md

```diff
@@ -0,0 +1,3 @@
+To do this, open `Info.plist` in your application bundle and define a **URL Scheme** for your app, like: `com.okta.example`
+
+This defines your **redirect scheme**. Add `:/callback` to the scheme to get the full redirect URI (like `com.okta.example:/callback`), which you'll need in the following steps.
```

Comment:
This defines your **redirect scheme**. Add `:/callback` to the scheme to get the full redirect URI (like `com.okta.example:/callback`), which you need in the following steps.